docs/knowledge_base/tutorials/scenario_management_overview/step_04/index.md

```diff
@@ -0,0 +1,309 @@
+used for all the scenarios created for a particular week. 
+This special 'official' scenario is called the 'Primary' scenario in Taipy.
+
+Note that Cycles can be ignored entirely if the business problem has no time frequency. 
+
+## Advantages of Cycles:
+
+- **Time-Based Organization:** Cycles facilitate the organization and analysis of scenarios over specific time periods.
+
+- **Primary Scenario Identification:** Cycles allow the designation of a primary scenario for reference or official analysis.
+
+- **Enhanced Data Consistency:**
+   They ensure data and parameter consistency across scenarios linked to a specific time period.
+
+- **Custom Frequency:** Cycles can be customized to various frequencies to align with specific business needs.
+
+- **Scenario Comparison:** Cycles make it easier to compare and analyze different scenarios within the same time frame.
+
+## Example: Filtering by Month
+
+```python
+def filter_by_month(df, month):
+    df['Date'] = pd.to_datetime(df['Date']) 
+    df = df[df['Date'].dt.month == month]
+    return df
+```
+
+
+![](config.svg){ width=700 style="margin:auto;display:block;border: 4px solid rgb(210,210,210);border-radius:7px" }
+
+## Configuration
+
+!!! example "Configuration"
+
+    === "Taipy Studio"
+
+        ![](config_04.gif){ width=700 style="margin:auto;display:block;border: 4px solid rgb(210,210,210);border-radius:7px" }
+        
+        - Construct the configuration
+
+        - Add the frequency property for the scenario and put "MONTHLY:FREQUENCY" (DAYLY, WEEKLY, MONTHLY, YEARLY)
+        
+        - Load the new [configuration](../src/config_04.toml) in the code
+
+    === "Python configuration"
+
+        The configuration is the same as the last step except for the scenario and task configuration. A new parameter is added for the frequency.
+
+        ```python
+        from taipy.config import Frequency
+
+        historical_data_cfg = Config.configure_csv_data_node(id="historical_data",
+                                                             default_path="time_series.csv")
+
+        month_cfg =  Config.configure_data_node(id="month")
+        month_values_cfg =  Config.configure_data_node(id="month_values")
+
+        task_filter_cfg = Config.configure_task(id="filter_by_month",
+                                                     function=filter_by_month,
+                                                     input=[historical_data_cfg, month_cfg],
+                                                     output=month_values_cfg)
+
+
+        scenario_cfg = Config.configure_scenario(id="my_scenario",
+                                                            task_configs=[task_filter_cfg],
+                                                            frequency=Frequency.MONTHLY)
+
+        ```
+
+
+Since we have specified 
+`frequency=Frequency.MONTHLY`, the corresponding scenario when created is automatically attached to the correct period (month). 
```

Comment:
```suggestion
Since we have specified `frequency=Frequency.MONTHLY`, the corresponding scenario when
created, is automatically attached to the correct period (month). 
```